package.json

```diff
@@ -16,7 +16,8 @@
     "prettier": "^1.14.3",
     "tap-spec": "^4.1.1",
```

Comment:
There's a security warning coming from lodash, which I think originates from this outdated package. If this is a quick fix, can you handle while you're in here?